package Exceptions.Throw;
/*

public class TestThrow2 {
	
	 * CHECKED EXCEPTION
	 * If we throw a checked exception using throw keyword, 
	 * it is must to handle the exception using catch block or the method must declare it using throws declaration.
	 *  Every subclass of Error and RuntimeException is an unchecked exception in Java. 
	 *  A checked exception is everything else under the Throwable class.
	 
	  public static void method() throws FileNotFoundException {  
		  
	        FileReader file = new FileReader("C:\\Users\\Anurati\\Desktop\\abc.txt");  
	        BufferedReader fileInput = new BufferedReader(file);  
	  
	      
	        throw new FileNotFoundException();  
	      
	    }  
	    //main method  
	    public static void main(String args[]){  
	        try  
	        {  
	            method();  
	        }   
	        catch (FileNotFoundException e)   
	        {  
	            e.printStackTrace();  
	        }  
	        System.out.println("rest of the code...");    
	  }  

}

*/
